sklearn/feature_extraction/heterogeneous.py

```diff
@@ -0,0 +1,148 @@
+from scipy import sparse
+import numpy as np
+
+from ..base import BaseEstimator, TransformerMixin
+from ..pipeline import _fit_one_transformer, _fit_transform_one, _transform_one
+from ..externals.joblib import Parallel, delayed
+from ..externals.six import iteritems
+
+
+class ColumnTransformer(BaseEstimator, TransformerMixin):
+    """Applies transformers to columns of a dataframe / dict.
```

Comment:
That would be pretty easy with the FunctionTransformer https://github.com/scikit-learn/scikit-learn/pull/4798